src/modules/add-remove.js

```diff
@@ -0,0 +1,126 @@
+import NewTask from './createTask.js';
+import {
+  markItemAsComplete, deleteAllTasks, updateInput, makeInputDefault,
+} from './interactive-page.js';
+
+const taskContainer = document.querySelector('.task-container');
+const addNewTaskInput = document.getElementById('add-new-task');
+const enterBtn = document.querySelector('.enter');
+const clearAllBtn = document.getElementById('clear-btn');
+const refresh = document.querySelector('.refresh');
+class AddRemoveTask {
+  constructor() {
+    this.store = JSON.parse(localStorage.getItem('ToDoList')) || [];
+  }
+
+  resetIndex() {
+    let initialIndex = 1;
+    this.store.forEach((task) => {
+      task.index = initialIndex;
+      initialIndex += 1;
+    });
+    localStorage.setItem('ToDoList', JSON.stringify(this.store));
+  }
+
+  clearCompletedTasks(button) {
+    button.addEventListener('click', () => {
+      this.store = this.store.filter((task) => task.completed === false);
+      taskContainer.innerHTML = '';
+      this.store.forEach((task) => {
+        this.newTask(task.description, task.id);
+      });
+      localStorage.setItem('ToDoList', JSON.stringify(this.store));
+    });
+  }
+
+  deleteTask(button) {
+    button.addEventListener('click', () => {
+      this.store = this.store.filter(
+        (tasks) => button.parentElement.id !== tasks.id,
+      );
+      taskContainer.innerHTML = '';
+      this.store.forEach((task) => {
+        this.newTask(task.description, task.id);
+      });
+      localStorage.setItem('ToDoList', JSON.stringify(this.store));
+    });
+  }
+
+  newTask(task, id) {
+    taskContainer.innerHTML
+      += `
+      <div class="task">
+          <input type="checkbox" class="check">
+          <input id="${id}" class="new-task" type="text" value="${task}">
+          <span id="${id}">
+            <i class="delete fa-solid fa-trash-can"></i>
+          </span>
+      </div>
```

Comment:
- [ ] Here if you use index instead of id it must be very useful and the code should be clean.